UPGRADE.md

```diff
@@ -16,6 +16,11 @@ There is a list of all the repositories maintained by monorepo, changes in log b
 * [shopsys/product-feed-zbozi]
 
 ## [From 7.0.0-alpha4 to Unreleased]
+- make Elasticsearch work
+    - copy new `docker/conf/docker-compose.yml.dist` into your local `docker-compose.yml`
+    - run `docker-compose up -d`

```

Comment:
I believe it must be `docker-compose up -d --force-recreate` as there are changes in the `docker-compose.yml`

Reply:
But the already running containers do not require recreating. I've tested it with just  `docker-compose -d` without any problems.